internal/blockstore/commit_and_query.go

```diff
@@ -65,6 +65,8 @@ func (s *Store) Commit(block *types.Block) error {
 		return err
 	}
 
+	fileops.SyncDir(s.fileChunksDirPath)
+
```

Comment:
Why do we need to fsync the dir after every write? that is very expensive and has no relation to the bug. I would remove that.